packages/broker/src/plugins/consoleMetrics/ConsoleMetricsPlugin.ts

```diff
@@ -24,68 +25,17 @@ export class ConsoleMetricsPlugin extends Plugin<ConsoleMetricsPluginConfig> {
     async start(): Promise<void> {
         const metricsContext = (await (this.streamrClient!.getNode())).getMetricsContext()
         this.producer = metricsContext.createReportProducer((report: MetricsReport) => {
-            this.printReport(report)
+            // omit timestamp info as that is printed by the logger
+            const data = omit(report, 'period')
```

Comment:
Does this function handle gracefully the case in which `period` would not be present in `report` (for whatever reason)?